internal/pkg/validator/validator.go

```diff
@@ -56,19 +74,18 @@ func Validate(value interface{}, rules ...Validation) error {
 	return nil
 }
 
-func processValidateError(err validator.ValidationErrors) error {
+func processValidateError(err validator.ValidationErrors, translator ut.Translator, fieldName string) error {
 	result := utils.NewMultiError()
 	for _, e := range err {
-		// Remove struct name, first part
-		namespace := regexpcache.MustCompile(`^([^.]+\.)?(.*)$`).ReplaceAllString(e.Namespace(), `$2`)
-		// Hide nested fields
-		namespace = strings.ReplaceAll(namespace, `__nested__.`, ``)
-		result.Append(fmt.Errorf(
-			"key=\"%s\", value=\"%v\", failed \"%s\" validation",
-			namespace,
-			e.Value(),
-			e.ActualTag(),
-		))
+		if e.Namespace() != "" {
+			// Remove struct name, first part
+			fieldName = regexpcache.MustCompile(`^([^.]+\.)?(.*)$`).ReplaceAllString(e.Namespace(), `$2`)
+			// Hide nested fields
+			fieldName = strings.ReplaceAll(fieldName, `__nested__.`, ``)
+			result.Append(fmt.Errorf("%s", e.Translate(translator)))
```

Comment:
Tady se to ale začíná komplikovat, protože hláška po překladu (`e.Translate(translator)`) už obsahuje ten field name z `err.Namespace()`, takže to původní odstraňování "struct name" a "hiding nested fields" nejde asi snadno udělat. Muselo by se to nějak vypreparovat z té přeložené hlášky a nahradit v ní a nebo ještě přetížit a upravit přímo ten překlad.

![CleanShot 2022-01-14 at 19 51 35@2x](https://user-images.githubusercontent.com/215660/149569394-55e59e96-3597-43c5-bcd3-fb7186eb9bd3.jpg)

Reply:
Nestacilo by rucne pridat `namespace` pred kazdy field? Nazov fieldu je vzdy prvy.
Napr. `project.id is a required field`.

Reply:
A z toho `namespace` odstranis struct name a aj tie `Anonymous` (`__nested__`) casti.

Reply:
Aha 😃 zkusím. Díky

Reply:
Jo tak ne, to už jsem zkoušel v pátek. 😃🙈 Začne se to zdvojovat, protože ten field name je v namespace i v tý přeložený hlášce:
![CleanShot 2022-01-17 at 09 12 41](https://user-images.githubusercontent.com/215660/149732063-dbb94afd-9770-4a89-aa6a-fb7aea8a5284.jpg)

Reply:
preberieme na standupe ;)